listener/src/event_handler/auction_settlement_handler.rs

```diff
@@ -9,7 +9,7 @@ use dfusion_core::models::Solution;
 
 use failure::Error;
 
-use graph::components::ethereum::EthereumBlock;
+use graph::components::ethereum::LightEthereumBlock;
```

Comment:
Whats the difference here? Was this a renaming on their side, or is this a change on our side?

Reply:
It was a renaming on their side.